crawl-ref/source/aptitudes.h

```diff
@@ -1253,6 +1253,45 @@ static const species_skill_aptitude species_skill_aptitudes[] =
     APT(SP_DEEP_DWARF,      SK_INVOCATIONS,     3),
     APT(SP_DEEP_DWARF,      SK_EVOCATIONS,      3),
 
+    // SP_DUSK_GIANT
+    APT(SP_DUSK_GIANT,      SK_FIGHTING,       -1),
+    APT(SP_DUSK_GIANT,      SK_SHORT_BLADES,   -2),
+    APT(SP_DUSK_GIANT,      SK_LONG_BLADES,    -2),
+    APT(SP_DUSK_GIANT,      SK_AXES,           -1),
+    APT(SP_DUSK_GIANT,      SK_MACES_FLAILS,   -1),
+    APT(SP_DUSK_GIANT,      SK_POLEARMS,       -1),
+    APT(SP_DUSK_GIANT,      SK_STAVES,          3),
+    APT(SP_DUSK_GIANT,      SK_SLINGS,          0),
+    APT(SP_DUSK_GIANT,      SK_BOWS,           -3),
+    APT(SP_DUSK_GIANT,      SK_CROSSBOWS,      -1),
+    APT(SP_DUSK_GIANT,      SK_THROWING,       -2),
+    APT(SP_DUSK_GIANT,      SK_ARMOUR,          0),
+    APT(SP_DUSK_GIANT,      SK_DODGING,         0),
+    APT(SP_DUSK_GIANT,      SK_STEALTH,         5),
+    #if TAG_MAJOR_VERSION == 34
+    APT(SP_DUSK_GIANT,      SK_STABBING,      UNUSABLE_SKILL),
+    #endif
+    APT(SP_DUSK_GIANT,      SK_SHIELDS,         -1),
+    #if TAG_MAJOR_VERSION == 34
+    APT(SP_DUSK_GIANT,      SK_TRAPS,         UNUSABLE_SKILL),
+    #endif
+    APT(SP_DUSK_GIANT,      SK_UNARMED_COMBAT,  0),
+    APT(SP_DUSK_GIANT,      SK_SPELLCASTING,   -1),
+    APT(SP_DUSK_GIANT,      SK_CONJURATIONS,   -1),
+    APT(SP_DUSK_GIANT,      SK_HEXES,          -3),
+    APT(SP_DUSK_GIANT,      SK_CHARMS,         -1),
+    APT(SP_DUSK_GIANT,      SK_SUMMONINGS,     -2),
+    APT(SP_DUSK_GIANT,      SK_NECROMANCY,     -1),
+    APT(SP_DUSK_GIANT,      SK_TRANSLOCATIONS, -1),
+    APT(SP_DUSK_GIANT,      SK_TRANSMUTATIONS, -1),
+    APT(SP_DUSK_GIANT,      SK_FIRE_MAGIC,     -2),
+    APT(SP_DUSK_GIANT,      SK_ICE_MAGIC,       1),
+    APT(SP_DUSK_GIANT,      SK_AIR_MAGIC,      -1),
+    APT(SP_DUSK_GIANT,      SK_EARTH_MAGIC,    -1),
+    APT(SP_DUSK_GIANT,      SK_POISON_MAGIC,    0),
+    APT(SP_DUSK_GIANT,      SK_INVOCATIONS,    -3),
+    APT(SP_DUSK_GIANT,      SK_EVOCATIONS,      0),
```

Comment:
Bad invo is all well and good, but -3 is too bad imo. It just encourages playing one of the gods that doesn't require invocations a bit too much, and puts high invo gods like Makhleb, Nemelex, Qazlal, Zin out of the question. -2 is just about passable, but -1 is the worst we have at the moment, for this reason.